webpack.config.ts

```diff
@@ -1,26 +1,26 @@
-import * as path from 'path';
-import OptimizeCssAssetsPlugin from 'optimize-css-assets-webpack-plugin';
-import MiniCssExtractPlugin from 'mini-css-extract-plugin';
-import HtmlWebPackPlugin from 'html-webpack-plugin';
-import { CleanWebpackPlugin } from 'clean-webpack-plugin';
-import { Configuration, HotModuleReplacementPlugin } from 'webpack';
+import * as path from "path";
```

Comment:
can you reformat this

Reply:
Oh prettier runs only in src, good catch.